app/uk/gov/hmrc/exports/migrations/MigrationRoutine.scala

```diff
@@ -39,13 +37,8 @@ class MigrationRoutine @Inject()(appConfig: AppConfig, exportsMigrationConfig: E
   private val db = client.getDatabase(uri.getDatabase)
 
   def execute(): Future[Unit] = Future {
-    if (exportsMigrationConfig.isExportsMigrationEnabled) {
-      logger.info("Exports Migration feature enabled. Starting migration with ExportsMigrationTool")
-      migrateWithExportsMigrationTool()
-    } else {
-      logger.info("Exports Migration feature disabled. Starting migration with Mongock")
-      migrateWithMongock()
-    }
+    logger.info("Exports Migration feature enabled. Starting migration with ExportsMigrationTool")
```

Comment:
Nit: since we are removing the feature flag, the message we log might only be `Starting migration with ExportsMigrationTool`